runtime/jaxrs/src/test/java/com/flit/runtime/jaxrs/FlitExceptionMapperTest.java

```diff
@@ -0,0 +1,43 @@
+package com.flit.runtime.jaxrs;
+
+import static org.junit.Assert.assertEquals;
+
+import com.flit.runtime.ErrorCode;
+import com.flit.runtime.FlitException;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+public class FlitExceptionMapperTest {
+
+  @Mock
+  private HttpServletRequest request;
+
+  @InjectMocks
+  private FlitExceptionMapper flitExceptionMapper = new FlitExceptionMapper();
+
+  @Before
+  public void setup() {
+    MockitoAnnotations.initMocks(this);
+  }
+
+  @Test
+  public void testToResponse() {
+    FlitException flit = FlitException.builder()
+        .withCause(new RuntimeException("Failed"))
+        .withErrorCode(ErrorCode.INTERNAL)
+        .withMessage("with this message")
+        .build();
+    Response response = flitExceptionMapper.toResponse(flit);
+    assertEquals(response.getStatus(), 500);
```

Comment:
**Thought**: Jaxrs provides response statuses. Would you consider using something like `Response.Status.INTERNAL_SERVER_ERROR.getStatusCode()` ?
Btw, I think there are tradeoffs with each option: more verbose and readable vs knowing the status code for what it represents.
While I think it is obvious for 500, it might not be for other status codes.
This is just thinking loud, interested in what you think.